### PR TITLE
Documentation website

### PR DESCRIPTION
**Pull request by bcalford**
In response to #98, creates a documentation website using Sphinx.

Currently being left as a draft as it is still rather bare bones and not yet live.

### CODE REVIEW

**Comment by bcalford** [comment]
Inside the docs folder, I have set up a documentation website that has all of the information from the README file as well as some extra development information from #96.

**Comment by christian-oreilly** [comment]
Looks great, @bcalford. However, we probably want to remove from GIT the `docs/build` directory as we generally don't version-control compiled stuff. Could you remove this folder (and its content) from GIT and push it again? It will make the code much easier to review because it will include only the written code, not what sphynx generates. You can also add this folder to the `.gitignore` file.

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""Configuration file for the Sphinx documentation builder."""
+
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "IPyNiiVue"
+project_copyright = "2024, Author Names"
+author = "Author Names"
+release = "2.0.1"
```
Comment (christian-oreilly) [inline docs/source/conf.py:10]:
```suggestion
project_copyright = "2024, Jan-Hendrik Müller, Trevor Manz, Bradley Alford, Anthony Androulakis, Taylor Hanayik, Christian O'Reilly"
author = "Jan-Hendrik Müller, Trevor Manz, Bradley Alford, Anthony Androulakis, Taylor Hanayik, Christian O'Reilly"
release = "2.1.0"
```

File: docs/source/conf.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+"""Configuration file for the Sphinx documentation builder."""
+
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "IPyNiiVue"
+project_copyright = "2024, Jan-Hendrik Müller, Trevor Manz, Bradley Alford, Anthony Androulakis, Taylor Hanayik, Christian O'Reilly"
+author = "Jan-Hendrik Müller, Trevor Manz, Bradley Alford, Anthony Androulakis, Taylor Hanayik, Christian O'Reilly"
```
Comment (christian-oreilly) [inline docs/source/conf.py:9]:
```suggestion
author = "Jan-Hendrik Müller, Trevor Manz, Bradley Alford, Anthony Androulakis, "
author += "Taylor Hanayik, Christian O'Reilly"
project_copyright = "2024, " + author
```

**Review by christian-oreilly — APPROVED** [review]

**Pull request merged**